easy-deploy-script/easy-deploy-viral-ngs.sh

```diff
@@ -174,13 +177,11 @@ function install_miniconda(){
         else
             miniconda_url=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
         fi
+        wget --progress=dot $miniconda_url -O "$INSTALL_PATH/Miniconda3-latest-x86_64.sh"
```

Comment:
`--progress=dot` is still verbose (see this [log](https://travis-ci.org/broadinstitute/viral-ngs-deploy/jobs/248959350)), too verbose for Travis.